packages/core/chrome/core-chrome-browser-internal/src/ui/header/screen_reader_a11y.tsx

```diff
@@ -27,7 +26,7 @@ export const ScreenReaderRouteAnnouncements: FC<{
       const breadcrumbText: string[] = [];
 
       // Reverse the breadcrumb title order and ensure we only pick up valid strings
-      fallbackBreadcrumbs.reverse().forEach((breadcrumb) => {
+      fallbackBreadcrumbs.reverse().map((breadcrumb) => {
```

Comment:
There's no point changing this to a `.map()` vs a `forEach()` because nothing needs to be returned here (which is the primary purpose of a map). `forEach` is more correct.

```suggestion
      fallbackBreadcrumbs.reverse().forEach((breadcrumb) => {
```